components/intro-section/intro-description.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styled from "styled-components";
+
+const DescriptionStyle = styled.div`
+  a {
+    text-decoration: none;
+    -webkit-transition: 0.2s;
+    -moz-transition: 0.2s;
+    -o-transition: 0.2s;
+    transition: 0.2s;
+    cursor: pointer;
+    color: #e0e0e0;
+    border-bottom: 1px solid #e0e0e0;
+    padding-bottom: 0;
+  }
+
+  a:hover {
+    border-bottom: 1px solid #0000ff;
```

Comment:
You just need to change the color, so I suggest just write `border-color: #0000ff;`

Reply:
does not work in chrome. Mark this as solved